feat(backend/admin):  enable backups and backup management for hcp clusters (#6261)
Azure/ARO-HCP: test/testdata/zz_fixture_TestMainListSuitesForEachSuite_dev_cd_check_paralleldev_cd_check_parallel.txt

Customer should be able to successfully upgrade control plane minor version from 5.11 minor to 5.12 minor
Customer should be able to create a cluster with an external auth config and get the external auth config
Customer should be able to lifecycle and confirm external auth on a cluster

> Backups can be created on-demand for an HCP cluster

Reviewer: ```suggestion
SRE can create backups on-demand for an HCP cluster
```

> Backups schedule pause stops backup execution for an HCP cluster

Reviewer: ```suggestion
SRE can pause or stops backup execution for an HCP cluster
```
